source/django-get-started/django-create-app.txt

```diff
@@ -0,0 +1,252 @@
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Recent Movies</title>
+         </head>
+         <body>
+            <h1>Five Most Recent Movies</h1>
+            <ul>
+               {% for movie in movies %}
+                     <li>
+                        <strong>{{ movie.title }}</strong> (Released: {{ movie.released }})
+                     </li>
+               {% empty %}
+                     <li>No movies found.</li>
+               {% endfor %}
+            </ul>
+         </body>
+         </html>
+      
+      This template formats the movie data requested by the ``recent_movies`` view.
+
+      Create another file in the ``sample_mflix/templates`` directory called
+      ``viewers_list.html`` and paste the following code:
+
+      .. code-block:: python
+
+         <!-- templates/viewers_list.html -->
+         <!DOCTYPE html>
+         <html lang="en">
+         <head>
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Viewers List</title>
+         </head>
+         <body>
+            <h1>Alphabetical Viewers List</h1>
+            <table>
+               <thead>
+                     <tr>
+                        <th>Name</th>
+                        <th>Email</th>
+                     </tr>
+               </thead>
+               <tbody>
+                     {% for viewer in viewers %}
+                        <tr>
+                           <td>{{ viewer.name }}</td>
+                           <td>{{ viewer.email }}</td>
+                        </tr>
+                     {% empty %}
+                        <tr>
+                           <td colspan="2">No viewer found.</td>
+                        </tr>
+                     {% endfor %}
+               </tbody>
+            </table>
+         </body>
+         </html>
+
+      This template formats the user data requested by the ``viewers_list`` view.
+
+   .. step:: Include your app in your project
+
+      Open the ``settings.py`` file in ``quickstart`` and edit your 
+      ``INSTALLED_APPS`` setting to resemble the following code:
+      
+      .. code-block:: python
+
+         INSTALLED_APPS = [
+             'sample_mflix.apps.SampleMflixConfig',
+             'quickstart.apps.MongoAdminConfig',
+             'quickstart.apps.MongoAuthConfig',
+             'quickstart.apps.MongoContentTypesConfig',
+             'django.contrib.sessions',
+             'django.contrib.messages',
+             'django.contrib.staticfiles',
+         ]
+
+   .. step:: Create migrations for your new models
+
+      From your project root, run the following command to create
+      migrations for the ``Movie``, ``Award``, and ``Viewer`` models and apply
+      the changes to the database:
+      
+      .. code-block:: bash
+
+         python3 manage.py makemigrations sample_mflix
```

Comment:
sample_mflix is an existing database, so the models should have an inner `Meta` class with [`managed = False`](https://docs.djangoproject.com/en/dev/ref/models/options/#managed) so that Django doesn't try to create the collections. Otherwise, I believe MongoDB will error with "collection already exists) at https://github.com/mongodb-labs/django-mongodb-backend/blob/eae01a45820de3b3a0efab4c1e85faa96335b4ad/django_mongodb_backend/schema.py#L25.